int32 number_of_detections
string[] names_detected
float32[] rectangles
